refract-spec.md

```diff
@@ -290,39 +293,37 @@ In JSON, an example of an object is:
 }
 ```
 
-In Refract, this object MAY Be expanded to include Member Elements as its content.
+In Refract, this object MAY Be expanded to include an array of elements as its content.
 
 ```json
 {
   "element": "object",
   "content": [
     {
-      "element": "member",
-      "content": {
-        "key": {
-          "element": "string",
-          "content": "foo"
-        },
-        "value": {
-          "element": "string",
-          "content": "bar"
-        }
-      }
+      "element": "string",
+      "meta": {
+        "name": "foo"
+      },
+      "content": "bar"
     }
   ]
 }
 ```
 
-### Member Element (Element)
+### Property Element (Element)
```

Comment:
Nit: It feels like we can get of this "Element" completely and just elaborate on the `name` property of `meta` no?